docs/source/user_guide/installation/windows-wsl.rst

```diff
@@ -1,108 +1,42 @@
 Windows Subsystem for Linux (WSL)
-======================================
+=================================
 
-We recommend the use of Windows Subsystem for Linux (WSL) to install
-PyBaMM, see the instructions below to get PyBaMM working using Windows,
-WSL and VSCode.
-
-.. contents::
+To make it easier to install PyBaMM, we recommend using the Windows Subsystem for Linux (WSL) along with Visual Studio Code. This guide will walk you through the process.
 
 Install WSL
 -----------
 
-Follow the instructions from Microsoft
-`here <https://docs.microsoft.com/en-us/windows/wsl/install-win10>`__.
-When given the option, choose the Ubuntu 18.04 LTS distribution to
-install. Best practices for setting up a WSL development environment can be found
-`here <https://docs.microsoft.com/en-us/windows/wsl/setup/environment>`__.
+Install Ubuntu 22.04 or 20.04 LTS as a distribution for WSL following `Microsoft's guide to install WSL <https://docs.microsoft.com/en-us/windows/wsl/install-win10>`__. For a seamless development environment, refer to `this guide <https://docs.microsoft.com/en-us/windows/wsl/setup/environment>`__.
 
 Install PyBaMM
 --------------
 
-Open a terminal window in your installed Ubuntu distribution by
-selecting “Ubuntu” from the start menu. This should give you a bash
-prompt in your home directory.
+Get PyBaMM's Source Code
+~~~~~~~~~~~~~~~~~~~~~~~~
 
-To download the PyBaMM source code, you first need to install git, which
-you can do by typing
+1. Open a terminal in your Ubuntu distribution by selecting "Ubuntu" from the Start menu. You'll get a bash prompt in your home directory.
 
-.. code:: bash
+2. Install Git by typing the following command:
 
-   sudo apt install git-core
+.. code:: bash
+     sudo apt install git-core
 
-For easier integration with WSL, we recommend that you install PyBaMM in
-your *Windows* Documents folder, for example by first navigating to
+3. Clone the PyBaMM repository. To install PyBaMM in your Windows Documents folder, navigate to it using the command::
 
 .. code:: bash
+     cd /mnt/c/Users/USER_NAME/Documents
```

Comment:
Is this required?

Reply:
No it is not, path doesn't affect performance in `WSL` now.